### PR TITLE
12111219梁瑞玮

### PR DESCRIPTION
**Pull request by AddisonX**

### CODE REVIEW

**Review by Claydon-Wang — COMMENTED** [review]
You can redefine the loss function by adjusting the hyperparameters, instead of creating a new class.

**Comment by AddisonX** [comment]
the dataset in the other class was adjusted



---Original---
From: ***@***.***&gt;
Sent at: 2023年11月7日(Tue) Evening6:37
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [SustechSTA303/STA303-Assignment01] 12111219梁瑞玮 (PR #59)




 
@Claydon-Wang commented on this pull request.
 
You can redefine the loss function by adjusting the hyperparameters, instead of creating a new class.
